src/scss/components/_footer.scss

```diff
@@ -46,6 +51,7 @@
 
     p,
     address {
+      color: get-color((color: air, opacity: .5));
```

Comment:
Really? :(

Reply:
@GeorgeJegon we are using [sass-lint](https://github.com/sasstools/sass-lint) instead [scss-lint](https://github.com/brigade/scss-lint). HoundCI uses the last one.

We are planning to move from sass-lint, because practically all tools uses scss-lint 😭